site/components/pages/Retirement/SingleRetirement/index.tsx

```diff
@@ -50,39 +71,87 @@ export const SingleRetirementPage: NextPage<Props> = (props) => {
       <Navigation activePage="Home" />
 
       <Section variant="gray" className={styles.section}>
-        <div className={styles.retirementContainer}>
-          <div className={styles.retirement_textGroup}>
-            <Text t="h2" as="h2" align="center">
-              Retirement SINGLE
-            </Text>
-            <Text align="center">timestamp: {retirement.timestamp}</Text>
-            <Text align="center">
-              tokenAddress: {retirement.offset.tokenAddress}
-            </Text>
-            <Text align="center">amount: {retirement.amount}</Text>
-            <Text align="center">
-              typeOfToken: {retirementIndexInfo.typeOfToken}
-            </Text>
-            <Text align="center">
-              beneficiaryName: {retirement.beneficiary}
-            </Text>
-            <Text align="center">
-              beneficiaryAddress: {retirement.beneficiaryAddress}
-            </Text>
-            <Text align="center">message: {retirement.retirementMessage}</Text>
-            <Text align="center">
-              View on{" "}
-              <a
-                target="_blank"
-                rel="noopener noreferrer"
-                href={`https://polygonscan.com/tx/${retirement.transaction.id}`}
-              >
-                polygonscan.com
-              </a>
-            </Text>
+        <RetirementHeader
+          title={`
+            ${retireData.amount} ${retireData.tokenLabel} ${t({
+            id: "retirement.single.header.offset",
+            message: "Offset",
+          })}`}
+          subline={retireData.beneficiaryName}
+        />
```

Comment:
Use lingui interpolation rather than plain JS. This should make it possible for translators to arrange the data however they see fit.

```suggestion
        <RetirementHeader
          title={t({
            id: "retirement.single.header.offset",
            message: `${retireData.amount} ${retireData.tokenLabel} Offset`,
          })}
          subline={retireData.beneficiaryName}
        />
```

Reply:
Done